lfs/lfs.go

```diff
@@ -207,3 +232,17 @@ func traceHttpReq(req *http.Request) string {
 func AllObjects() []localstorage.Object {
 	return objects.AllObjects()
 }
+
+func IfNoLocalObjExistsLinkOrCopyFromReferenceMedia(oid string, size int64) error {
+	if !ObjectExistsOfSize(oid, size) {
+		altMediafile := LocalReferencePath(oid)
+		mediafile, err := LocalMediaPath(oid)
+		if err != nil {
+			return err
+		}
+		if altMediafile != "" && FileExistsOfSize(altMediafile, size) {
+			return LinkOrCopy(altMediafile, mediafile)
+		}
+	}
+	return nil
```

Comment:
I tend to prefer smaller guard clauses. What about:

```go
func IfNoLocalObjExistsLinkOrCopyFromReferenceMedia(oid string, size int64) error {
    if ObjectExistsOfSize(oid, size) {
        return nil
    }

    // rest of the function
}
```

Also, I'm not thrilled with the name `IfNoLocalObjExistsLinkOrCopyFromReferenceMedia`. What about just `LinkOrCopyFromReference` ?